firefox-ios/firefox-ios-tests/Tests/XCUITests/DownloadsTests.swift

```diff
@@ -16,7 +16,7 @@ class DownloadsTests: BaseTestCase {
     override func tearDown() {
         // The downloaded file has to be removed between tests
         app.terminate()
-        app.activate()
```

Comment:
@isabelrios This line is the change that fixes the UI test failure

Reply:
As commented on slack, the test is failing only in this PR and in this scneario for some reason that we don't understand. It is fine to change this to unblock the PR, although it may be also risky since we don't know the cause and if this could appear at some point while using the app.